modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -1,5 +1,5 @@
 :description: This page gives an introduction to database administration using Neo4j.
-:page-aliases: manage-databases/introduction.adoc, manage-databases/index.adoc
+:page-aliases: manage-databases/introduction.adoc, manage-databases/index.adoc, manage-databases/introduction.adoc#manage-databases-default
```

Comment:
I tried with anchors, but it didn't work. Have you tested this one?

Reply:
I would suggest that we update the links in all those manuals. It's a cleaner solution.

Reply:
Actually, it's a redirect thing. We need to redirect the `manage-databases/introduction` to `database-administration`.

Reply:
Let me know if you want me to add it or you'll do it.

Reply:
Yes but we are already redirecting `manage-databases/introduction` to `database-administration`, the thing is that in Bloom and other drivers manuals the link includes an anchor. I tried the redirect with the anchor locally and it worked, but I agree that changing in the other manuals might be a cleaner option!

Reply:
No, we don't. There is no redirect, just a page-aliase.  Page-aliases do not span across manuals.

Reply:
We need to create a redirect anyway, as every page-alias must be backed by a redirect. Then, those links will work as they will be redirected to the proper page.

Reply:
How do you want to create the redirect, then? I thought we were supposed to use page-aliases

Reply:
The redirects are created in the manual-modeling-antora repo - see the details here: https://github.com/neo-technology/neo4j-manual-modeling-antora/wiki/Redirects

Reply:
I'll close this PR then and work on the redirects. Thanks!